assets/js/components/BatchEdit/About/RightsMetadata.jsx

```diff
@@ -2,7 +2,10 @@ import React from "react";
 import PropTypes from "prop-types";
 import UIFormBatchFieldArray from "../../UI/Form/BatchFieldArray";
 import UIFormSelect from "../../UI/Form/Select";
-import { RIGHTS_METADATA } from "../../../services/metadata";
+import {
+  RIGHTS_METADATA,
+  getCodedTermSelectOptions,
+} from "../../../services/metadata";
```

Comment:
Can we make this, and all future imports relative imports to save us the work down the road?   `import from "@js/services....."`